README.md

```diff
@@ -69,9 +76,36 @@ report. Keep this in mind when setting the user roles which can access each repo
   * The API key has to be sent as the value of the `api_key` field in a HTTP POST request.
   * Administrators have the option (in the module system settings) to prohibit non-administrators
     from saving reports where API access is enabled.
+* **Allow public access** \* determines whether the report can be accessed publicly (without a
+  REDCap login)
+  * If the report can be downloaded or retrieved as image, this can be done by all users who access
+    the report at the public URL.
+  * Administrators have the option (in the module system settings) to prohibit non-administrators
+    from saving reports where public access is enabled.
 
 \* only available on some report types
 
+### Accumulation report options
```

Comment:
Google AI calls 'An accumulation report is a report about a collection of waste or rubbish that has been dumped in an undesirable location.'  Even though I was aware in advance of what this report type is, the name and description could be made a little clearer.  It all sounds rather techy at  the moment.  How about putting something like this near the top of the description: Accumulation report will show increasing counts such as participant recruits per site over time in a clinical trial.

Reply:
I'm not sure I would consider Google AI reliable, but I have amended the documentation as you have suggested.

Reply:
I wouldn't consider Google AI reliable either... but this is for everyone, and more clarity never hurts.  Thanks for updating!